libs/sdk/src/skill/__tests__/skill-audit.helper.spec.ts

```
@@ -0,0 +1,254 @@
+  });
+
+  it('does NOT call eval or require — works in a sandbox without require', () => {
+    // Hide require in this scope. A passing test confirms the helper
+    // doesn't transitively touch `require`/`eval('require')` on the hot
+    // path. (The injected factory IS where the host wires the actual
+    // module, so it never traverses require here.)
+    setSkillAuditFactory(() => fakeModule);
+    const logger = makeLogger();
+    const providers = makeProviders();
+    // Wrap the call in a sentinel that would trip if `eval` was used.
+    const originalEval = globalThis.eval;
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)